src/autoscaler/metricsgateway/cmd/metricsgateway/main.go

```diff
@@ -50,8 +48,8 @@ func main() {
 		_, _ = fmt.Fprintf(os.Stdout, "%s\n", err.Error())
 		os.Exit(1)
 	}
-	//nolint:staticcheck  // SA1019 TODO: https://github.com/cloudfoundry/app-autoscaler-release/issues/548
-	metricServerClientTLSConfig, err := cfhttp.NewTLSConfig(conf.Emitter.MetricsServerClientTLS.CertFile, conf.Emitter.MetricsServerClientTLS.KeyFile, conf.Emitter.MetricsServerClientTLS.CACertFile)
+
+	metricServerClientTLSConfig, err := conf.Emitter.MetricsServerClientTLS.CreateClientConfig()
 	if err != nil {
 		_, _ = fmt.Fprintf(os.Stdout, "%s\n", err.Error())
```

Comment:
Cosmetic thing: As we already have a logger, we could probably use that instead of the Fprintfs. But we can take that in another PR.

Reply:
Thats part of another issue ... refactoring the mains